source/docs/installation/index.rst

```diff
@@ -0,0 +1,51 @@
+:orphan:
+
+Getting Started
```

Comment:
I'm not sure what the point of this page is if the only way you can get to it is by knowing to click the "Installation" button on the home page instead of using the sidebar navigation.